PathKit/PathKit/PathKit.h

```diff
@@ -0,0 +1,19 @@
+//
+//  PathKit.h
+//  PathKit
+//
+//  Created by Jeremie Benhamron on 2019-01-22.
+//  Copyright © 2019 jabt. All rights reserved.
+//
```

Comment:
Let's use an open-source license and set that as the default for new files.

Reply:
Ok will do tonight